cadquery/cq.py

```diff
@@ -1916,6 +1916,41 @@ def makeCircleWire(obj):
 
         return self.eachpoint(makeCircleWire, useLocalCoordinates=True)
 
+    # ellipse from current point
+    def ellipse(self, x_radius, y_radius, angle1=360, angle2=360, rotation_angle=0.0,
+                closed=False, forConstruction=False):
+        """
+        Make an ellipse for each item on the stack.
+        :param x_radius: x radius of the ellipse (x-axis of plane the ellipse should lie in)
+        :type x_radius: float > 0
+        :param y_radius: y radius of the ellipse (y-axis of plane the ellipse should lie in)
+        :type y_radius: float > 0
+        :param forConstruction: should the new wires be reference geometry only?
+        :type forConstruction: true if the wires are for reference, false if they are creating
+            part geometry
+        :param rotation_angle: angle to rotate the ellipse (0 = no rotation = default)
+        :type rotation_angle: float
+        :param angle1: start angle of arc
+        :type angle1: float
+        :param angle2: end angle of arc
+        :type angle2: float
+        :param rotation_angle: angle to rotate the created ellipse / arc
+        :type rotation_angle: float
+        :return: a new CQ object with the created wires on the stack
+        
+        *NOTE* Due to a bug in opencascade (https://tracker.dev.opencascade.org/view.php?id=31290)
```

Comment:
Is this true? I thought that ellipse is added to `pendingWires` which are not used to calculate center.

Reply:
```python
v = (cq.Workplane("XY")
     .center(10, 20).ellipse(100,10)
     .ellipse(50, 5).vertices()).objects[0]
v.X, v.Y
```
results in `(58.34650359492122, 20.0)`

While 
```python
v = (cq.Workplane("XY")
     .center(10, 20).ellipse(100,10)
     .center(0,0).ellipse(50, 5).vertices()).objects[0]
v.X, v.Y
```

gets the correct result `(60.0, 20.0)`. This is btw. a test case I've added.

If you look at the `eachpoint` code
https://github.com/CadQuery/cadquery/blob/74573fc3bb032321c50a9e817dbf806c2a286d53/cadquery/cq.py#L1825-L1834
it actually uses `Center` and `circle` and `ellipse`  use `eachpoint`.

Reply:
May bad, the wire is added to pendingWires and objects